packages/connect-iframe/webpack/sessions.webpack.config.ts

```diff
@@ -0,0 +1,33 @@
+import path from 'path';
+import webpack from 'webpack';
+import merge from 'webpack-merge';
+
+import { config as baseConfig } from './base.webpack.config';
+import { getSharedworkerDistPathForProject } from './utils';
+
+const project = process.env.PROJECT || 'iframe';
```

Comment:
do we need to specify project anymore? I mean - this should be only deployed with connect now so my first intuition is that accessing env.PROJECT might not be needed.

Reply:
@karliatto? what do you think? is it needed?